### PR TITLE
removing slow passes

### PR DESCRIPTION
**Pull request by 1sanz**
- loc_program, loc_main and reorder_commented to check correctness

### CODE REVIEW

File: tile/targets/gpu/amd.jsonnet [hunk 1]
```diff
@@ -31,24 +31,25 @@ local PARAMS = {
...
-              name: 'loc_main',
-              pass: {
-                '@type': 'type.vertex.ai/vertexai.tile.codegen.proto.LocateMemoryPass',
-                reqs: ['main'],
-                loc: { devs: [{ name: 'GLOBAL', units: [{ offset: 0 }] }] },
-              },
-            },
-
+            
+            // {
```
Comment (flaub) [inline tile/targets/gpu/amd.jsonnet]: Let's remove these lines if we don't need them anymore (we can always recover them via git history)

File: tile/targets/gpu/cm.jsonnet [hunk 2]
```diff
@@ -19,23 +19,24 @@ local PARAMS = {
           // Define the stripe passes
```
Comment (flaub) [inline tile/targets/gpu/cm.jsonnet]: I'm thinking we should not touch the `cm` configuration until we have a way of running tests on the `cm` backend.

File: tile/targets/gpu/amd.jsonnet (hunk 1 above)
Comment (1sanz) [reply tile/targets/gpu/amd.jsonnet]: removed.

File: tile/targets/gpu/cm.jsonnet (hunk 2 above)
Comment (1sanz) [reply tile/targets/gpu/cm.jsonnet]: passes put back in

**Pull request merged**